tests/blocks/test_core.py

```diff
@@ -405,18 +406,25 @@ class SecretBlock(Block):
         assert isinstance(blockdoc.data["x"], SecretStr)
         assert isinstance(blockdoc.data["y"], SecretBytes)
 
-        json_blockdoc = json.loads(blockdoc.json())
+        json_blockdoc = blockdoc.model_dump(mode="json")
         assert json_blockdoc["data"] == {
-            "w": {
-                "Here's my shallow secret": "**********",
-                "deeper secrets": "**********",
-            },
+            "w": "**********",
```

Comment:
a pydantic `Secret[dict[str, Any]]` dumps the entire value like this by default, im not sure there's a reason we've historically exposed the keys of a `SecretDict`